airflow/providers/amazon/aws/sensors/athena.py

```diff
@@ -49,20 +52,19 @@ class AthenaSensor(BaseSensorOperator):
 
     @apply_defaults
     def __init__(self,
-                 query_execution_id,
-                 max_retires=None,
-                 aws_conn_id='aws_default',
-                 sleep_time=10,
-                 *args, **kwargs):
+                 query_execution_id: str,
+                 max_retires: Optional[int] = None,
+                 aws_conn_id: str = 'aws_default',
+                 sleep_time: int = 10,
+                 *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
         self.aws_conn_id = aws_conn_id
         self.query_execution_id = query_execution_id
         self.sleep_time = sleep_time
         self.max_retires = max_retires
-        self.hook = None
 
-    def poke(self, context):
-        state = self.get_hook().poll_query_status(self.query_execution_id, self.max_retires)
+    def poke(self, context: Dict[Any, Any]) -> bool:
```

Comment:
```suggestion
    def poke(self, context: dict) -> bool:
```

since we don't have any specific hints inside `Dict[Any, Any]`